public/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Allow: /
+Disallow: /*?
+
+Host: https://rs.school
```

Comment:
It looks like google doesn't support `Host` field: https://developers.google.com/search/docs/crawling-indexing/robots/robots_txt#syntax. Or am I missing something?

Reply:
That's right, this field is unnecessary here. I'll delete it